### PR TITLE
Feature/changelog and dod

### PR DESCRIPTION
**Pull request by danyball**
Fixes #(issue).

# Done is when (DoD):
- [ ] I have added UI/Unit tests for my changes
- [ ] I added modified component properties to the typescript typings
- [ ] Design has been reviewed with (here name of reviewer)
- [ ] My changes are accordingly to our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)
- [ ] Old tests and eslint rule are not broken
- [ ] My changes are well documented in the README and showcased in the stories 
- [ ] Tested with IE

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas (focus on **WHY**)
- [ ] I have made/updated the ChangeLog Section in the README

### CODE REVIEW

File: src/components/20-molecules/policy-features/CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
-# Changelog
```
Comment (01231) [inline src/components/20-molecules/policy-features/CHANGELOG.md:1]: Why don't we keep the header?

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -2,11 +2,12 @@ Fixes #(issue).
 
 # Done is when (DoD):
 - [ ] I have added UI/Unit tests for my changes
+- [ ] Old tests and eslint rule are not broken
```
Comment (raphaellueckl) [inline PULL_REQUEST_TEMPLATE.md:5]: This is already enforced by automation.

Comment (LucaMele) [reply PULL_REQUEST_TEMPLATE.md:5]: Still good to have i guess

**Review by LucaMele — COMMENTED** [review]
Please add it to create-a-m-o  once we decided which template

File: src/components/20-molecules/policy-features/CHANGELOG.md (hunk 1 above)
Comment (danyball) [reply src/components/20-molecules/policy-features/CHANGELOG.md:1]: No need, its the CHANGELOG file

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (danyball) [reply PULL_REQUEST_TEMPLATE.md:5]: This is not a new rule. i just put it up

**Comment by danyball** [comment]
I need a approve here ;)

**Review by LucaMele — APPROVED** [review]
LGTM

**Pull request merged**